src/main/java/pl/betoncraft/betonquest/compatibility/jobsreborn/Event_LeaveJob.java

```diff
@@ -20,39 +20,44 @@
 
 import com.gamingmesh.jobs.Jobs;
 import com.gamingmesh.jobs.container.Job;
+import com.gamingmesh.jobs.container.JobsPlayer;
 import org.bukkit.entity.Player;
 import pl.betoncraft.betonquest.Instruction;
 import pl.betoncraft.betonquest.api.QuestEvent;
 import pl.betoncraft.betonquest.exceptions.InstructionParseException;
 import pl.betoncraft.betonquest.utils.PlayerConverter;
 
 public class Event_LeaveJob extends QuestEvent {
-    private String sJobName;
+    private Job sJob;
```

Comment:
Should also be a String

Reply:
No, it makes no sense and it will decreases the performance because you iterating multiple for loops.